src/util.ts

```diff
@@ -88,3 +88,26 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * Returns the declarations for the given decorator.
```

Comment:
Could you move it to a file `decorators.ts`? It's better to have it in isolation than in a junkyard ;-)